### PR TITLE
rules added for OCP-36165

### PR DESCRIPTION
**Pull request by yapei**

### CODE REVIEW

File: lib/rules/web/admin_console/4.7/actions.xyaml [hunk 1]
```diff
@@ -153,8 +153,8 @@ confirm_cancel_action:
   action:
     if_param:
       cancel: 'false'
-    ref: click_not_cancel
-click_not_cancel:
+    ref: click_cancel_in_modal
```
Comment (yapei) [inline lib/rules/web/admin_console/4.7/actions.xyaml:156]: rename the action to be more readable

**Comment by yapei** [comment]
@liangxia Could you help review and merge if no objections?

**Review by pruan-rht — COMMENTED** [review]
/lgtm

**Pull request merged**